validphys2/src/validphys/process_options.py

```diff
@@ -351,7 +351,7 @@ def _dybosonpt_xq2map(kin_dict):
 DY_2L = _Process(
     "DY_2L",
     "DY W or Z -> 2 leptons ",
-    accepted_variables=(_Vars.y, _Vars.eta, _Vars.m_W2, _Vars.m_Z2, _Vars.sqrts),
+    accepted_variables=(_Vars.y, _Vars.abs_eta, _Vars.m_W2, _Vars.m_Z2, _Vars.sqrts),
```

Comment:
```suggestion
    accepted_variables=(_Vars.y, _Vars.eta, _Vars.abs_eta, _Vars.m_W2, _Vars.m_Z2, _Vars.sqrts),
```

the tests are failing since you remove this variable that some other datasets are using

Reply:
Thank you for pointing this out, I have just fixed this